dwave/system/samplers/dwave_sampler.py

```diff
@@ -475,3 +477,76 @@ def validate_anneal_schedule(self, anneal_schedule):
         for (t0, s0), (t1, s1) in zip(anneal_schedule, anneal_schedule[1:]):
             if round(abs((s0 - s1) / (t0 - t1)),10) > max_slope:
                 raise ValueError("the maximum slope cannot exceed {}".format(max_slope))
+
+
+    def to_networkx_graph(self, data=True, coordinates=False, 
```

Comment:
I would actually not allow _any_ parameters here. Especially since `coordinates` don't work currently and others like `offset_lists` do not apply.

`data` should default to True, otherwise I think coordinates should be handled "up stream", e.g. https://github.com/dwavesystems/dwave-system/issues/289